src/App.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import WebPlayback from './components/WebPlayBack';
+import PlayerUI from './components/PlayerUI'
```

Comment:
minor: This can be removed since we're not using the PlayerUI component in App.tsx

Reply:
Removed Import - was there for a PlayerUI Test and since test is gone no longer needed.